docs/dev/batch/hadoop_compatibility.md

```diff
@@ -192,9 +192,9 @@ The following example shows how to use Hadoop `Mapper` and `Reducer` functions.
 
 {% highlight java %}
 // Obtain data to process somehow.
-DataSet<Tuple2<Text, LongWritable>> text = [...]
+DataSet<Tuple2<LongWritable, Text>> text = [...]
 
-DataSet<Tuple2<Text, LongWritable>> result = text
+DataSet<Tuple2<LongWritable, Text>> result = text
```

Comment:
I think this line should not be changed.

Reply:
Good catch!
Thanks for the quick review.